modules/ossm-recommended-resources.adoc

```diff
@@ -55,6 +55,43 @@ spec:
             limits: {}
 ----
 +
+.. Define Kiali requests and limits under `kiali.deployment`.
++
+.Example version `kiali.deployment`
```

Comment:
In the previous sample, it is `.Example version {MaistraVersion} ServiceMeshControlPlane`. But what does actually mean? I think this is totally redundant.